cherrypy/_helper.py

```diff
@@ -82,54 +82,56 @@ def popargs(*args, **kwargs):
 
     This decorator may be used in one of two ways:
 
-    As a class decorator:
-    @cherrypy.popargs('year', 'month', 'day')
-    class Blog:
-        def index(self, year=None, month=None, day=None):
-            #Process the parameters here; any url like
-            #/, /2009, /2009/12, or /2009/12/31
-            #will fill in the appropriate parameters.
-
-        def create(self):
-            #This link will still be available at /create.  Defined functions
-            #take precedence over arguments.
-
-    Or as a member of a class:
-    class Blog:
-        _cp_dispatch = cherrypy.popargs('year', 'month', 'day')
-        #...
+    As a class decorator::
+
+        @cherrypy.popargs('year', 'month', 'day')
```

Comment:
I think that it may be a good idea to also explicitly specify the language for highlighting in code blocks. But that's minor. WDYT?

Reply:
I think it is totally worth it. There are some other warnings elsewhere in the documentation build that are related to code highlighting not working due to invalid syntax or something like that, so that should be cleaned up as well. But for this PR I'll at a minimum add Python to this block.

Reply:
Sure. It's a good idea to separate PRs so that they'd be atomic.